modules/folder/outputs.tf

```diff
@@ -24,7 +24,7 @@ output "firewall_policies" {
 output "firewall_policy_id" {
   description = "Map of firewall policy ids created in this folder."
   value = {
-    for name, _ in var.firewall_policies :
+    for name, _ in local.firewall_policies :
     name => google_compute_organization_security_policy.policy[name].id
```

Comment:
Wouldn't it make sense to cycle on the actual resource?

```hcl
{ for k, v in google_compute_organization_security_policy.policy : k => v.id }
```

Reply:
Deferred to the upcoming refactor from Julio